data/org.gnome.shell.gschema.xml.in.in

```diff
@@ -254,6 +254,69 @@ value here is from the GsmPresenceStatus enumeration.</_summary>
       <_summary>Keybinding that shows the force app exit dialog</_summary>
       <_description></_description>
     </key>
+    <key name="activate-first-icon" type="as">
```

Comment:
Would make the code a bit nicer, since you wouldn't need the "word" array. Up to you but I suggest changing it too.